packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -1115,7 +1115,6 @@ class RenderObjectToWidgetAdapter<T extends RenderObject> extends RenderObjectWi
       element._newWidget = this;
       element.markNeedsBuild();
     }
-    return element!;
```

Comment:
I don't think we can remove this one. The if/else branches do not have returns and rely on this statement to return the element.